cypress/helpers/cypressTasks/dynamo/dynamo-helpers.ts

```diff
@@ -94,6 +94,23 @@ export const getFeatureFlagValue = async ({
   return !!result?.Item?.current;
 };
 
+export const getRawFeatureFlagValue = async ({
+  flag,
+}: {
+  flag: string;
+}): Promise<boolean> => {
```

Comment:
I think we need to either
```suggestion
}): Promise<boolean | undefined> => {
```